server/athenian/api/internal/features/entries.py

```diff
@@ -1258,12 +1270,19 @@ async def batch_calc_jira_metrics_line_github(
         reporters, assignees, commenters = self._merge_jira_participants(
             [t.participants for request in requests for t in request.teams],
         )
+
+        jira_filters = [
+            td.jira_filter for td in chain.from_iterable(req.teams for req in requests)
+        ]
+        jira_filter = reduce(operator.or_, jira_filters)
```

Comment:
This code repeats three times, it makes sense to factor it out to `_union_jira_filters()` or smth like that.

Reply:
I'll need `jira_filters` and `jira_filter` later, anyway a refactored a little bit of repetition into `MetricsLineRequest`